sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ReactorSender.java

```diff
@@ -289,19 +408,29 @@ public void dispose() {
         tokenManager.close();
     }
 
-    Mono<Void> send(byte[] bytes, int arrayOffset, int messageFormat) {
+    Mono<Void> sendTransaction(byte[] bytes, int arrayOffset, int messageFormat, AmqpTransaction transactionId) {
```

Comment:
There are multiple overloads of sendTransaction. Can you group these methods together. Also, why is one private and the other ones package-private?

Reply:
This parameter is also "transaction" rather than transactionId